lib/morse_translator.rb

```diff
@@ -38,6 +38,63 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+
+    @morse_to_english = {
+                    ".-" => "a",
+                    "-..." => "b",
+                    "-.-." => "c",
+                    "-.." => "d",
+                    "." => "e",
+                    "..-." => "f",
+                    "--." => "g",
+  	                "...." => "h",
+  	                ".." => "i",
+  	                ".---" => "j",
+  	                "-.-" => "k",
+  	                ".-.." => "l",
+  	                "--" => "m",
+  	                "-." => "n",
+                   	"---" => "o",
+  	                ".--." => "p",
+  	                "--.-" => "q",
+  	                ".-." => "r",
+  	                "..." => "s",
+  	                "-" => "t",
+  	                "..-" => "u",
+  	                "...-" => "v",
+  	                ".--" => "w",
+  	                "-..-" => "x",
+  	                "-.--" => "y",
+  	                "--.." => "z",
+  	                "/" => " ",
+  	                ".----" => "1",
+  	                "..---" => "2",
+  	                "...--" => "3",
+  	                "....-" => "4",
+  	                "....." => "5",
+  	                "-...." => "6",
+  	                "--..." => "7",
+  	                "---.." => "8",
+  	                "----." => "9",
+  	                "-----" => "0"}
+
+      @joined_morse_code = []
+      @joined_english_code = []
+
   end
 
+    def translate(input)
+     input.downcase.chars.each do |letter|
+     @joined_morse_code <<  @dictionary[letter]
```

Comment:
You are changing each value and shoveling it into an array. In the future, recognize this as a map instead of an each